Monika-After-Story/MonikaModDev
# Adding white + navy blue dress

#4550 

# Key changes
* White and Navy Blue dress with bunny scrunchie from @Velius94 
* `bunny-scrunchie` ACS type with template. muxable with ribbons
* the bunny-scrunchie also only works with non-twintail ribbon hairs
* `_acs_wear_if_gifted` - only wears an ACS if it has been gifted/reacted to
* Exprops now will have constants in `mas_sprites`. we'll do this on a rolling basis. I need to update the wiki first.
* the hair tie selector now filters hair ties based on the current hair. This means no single-sided ACS (the bunny scrunchie, black bow) when wearing a twin-tail hair style
* ponytail-braid hair style no longer has the twintails prop

## New exprop
* `excluded-hair-props` - value should be a list of hair props that are in compatible with this ACS

# TODO
- [x] thumbs
- [x] select dlg if desired
- [x] dlg-desc
- [x] determine what the blue ribbon thing is. We can't really use `Ribbon (Blue)` because it looks different than the normal ribbons.

# Testing
* verify that the clothes + acs look good
* in outfit mode, the clothes will swap to a ribbon + non-twintails hair style to wear the bunny scrunchie.
* verify the bunny scrunchie and black bow **cannot** be worn with a twintail hair style
* verify the hair tie selector does not list bunny scrunchie or blackbow if a twintail hair style is worn
